### PR TITLE
Closes #2529 documentation add words to desription 2 functions

### LINKED ISSUE #2529: Documentation:  Add more keywords to descriptions of `derive_param_computed()` and `derive_vars_computed()`
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

A user gave feedback that they were looking to implement a derived parameter that was a ratio of two other parameters and they searched for the keyword "ratio" in the website searchbar. This did not redirect them to `derive_param_computed()`.

From a searchability perspective it could be beneficial to add a few of keywords to the description of these functions so that users may be redirected to them. Possible words:
- sum
- difference
- product/mutliplication
- division/ratio
- exponent(iation)
- logarithm

It's a little brute force-y but may be a quick and easy improvement.

### PR DESCRIPTION
**Pull request by jimrothstein**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Review the [Cheat Sheet](https://github.com/pharmaverse/admiral/blob/main/inst/cheatsheet/admiral_cheatsheet.pdf). Make any required updates to it by editing the file `inst/cheatsheet/admiral_cheatsheet.pptx` and re-upload a PDF and a PNG version of it to the same folder. (The PNG version can be created by taking a screenshot of the PDF version.)
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers). A Developer Notes section is available in `NEWS.md` for tracking developer-facing issues.  
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by jimrothstein** [comment]
@manciniedoardo 

`derive_param_computed`  Added few words to `description`.  Any suggestions/improvements?
( Then will do `derive_vars_computed` the same.)   Thx

File: R/derive_param_computed.R [hunk 1]
```diff
@@ -2,9 +2,12 @@
 #'
 #' Adds a parameter computed from the analysis value of other parameters. It is
 #' expected that the analysis value of the new parameter is defined by an
-#' expression using the analysis values of other parameters. For example mean
-#' arterial pressure (MAP) can be derived from systolic (SYSBP) and diastolic
-#' blood pressure (DIABP) with the formula
+#' expression using the analysis values of other parameters, such as addition/sum,
+#' subtraction/difference, multiply/product, division/ratio,
```
Comment (manciniedoardo) [inline R/derive_param_computed.R:6]:
```suggestion
#' subtraction/difference, multiplication/product, division/ratio,
```

File: R/derive_param_computed.R [hunk 2]
```diff
@@ -2,9 +2,12 @@
...
+#' exponentiate/logarithm, or by formula.
```
Comment (manciniedoardo) [inline R/derive_param_computed.R:7]:
```suggestion
#' exponentiation/logarithm, or by formula.
```

File: DESCRIPTION [hunk 3]
```diff
@@ -17,6 +17,8 @@ Authors@R: c(
     person("Sadchla", "Mascary", role = "aut"),
     person("Gordon", "Miller", role = "aut"),
     person("Lina", "Patil", role = "aut"),
+    person("Jim", "Rothstein", role = "aut",
```
Comment (manciniedoardo) [inline DESCRIPTION:20]: great to have you on board 😄

**Review by manciniedoardo — CHANGES_REQUESTED** [review]
Thanks @jimrothstein - a couple of comments but looks good.

**Comment by jimrothstein** [comment]
image

**Review by manciniedoardo — APPROVED** [review]
Thanks @jimrothstein - i think the failing link is not from this PR - we can merge 😄

**Pull request merged**